.github/actions/semver/README.md

```diff
@@ -0,0 +1,38 @@
+# Purpose
+
+This is github action with bash based script that provides semantic version generation with specific logic (different than gitversion).
+A unique release name is guaranteed for every branch+changeset combination.
+
+Semantic version based on tag. Tag choose priority according branch type
+
+- main branch releases can be versioned only from current commit tag
+- detached HEAD should have tag that belongs main branch
+- release/* releases can be versioned only from branch name but not tag, like release/1.1.1
+- hotfix releases based on main tag (or failover) and postfix as title for semver
+- all others in priority: branch tag -> main tag -> failover.
+That grants us option when we can build well versioned feature/* releases with any tag inside. 
+
+Image below is editable via draw.io
+
+![image](semver.png)
```

Comment:
Shouldn't image.png be at the same level as README.md file?

Reply:
Fixed